tests/integration-tests/tests/performance_tests/test_starccm.py

```diff
@@ -51,8 +57,16 @@ def test_starccm(
     number_of_nodes,
     test_datadir,
     scheduler_commands_factory,
+    s3_bucket_factory,
 ):
-    cluster_config = pcluster_config_reader(number_of_nodes=max(number_of_nodes))
+    # Create S3 bucket for custom actions scripts
+    bucket_name = s3_bucket_factory()
+    s3 = boto3.client("s3")
+    s3.upload_file(str(test_datadir / "dependencies.install.sh"), bucket_name, "scripts/dependencies.install.sh")
```

Comment:
to avoid the creation of an extra s3 bucket for a script of 1 line you can:
1. remove the `dependencies.install.sh` from the `OnNodeConfigured` and instead use the `remote_command_executor.run_remote_script` approach.
2. simply use `remote_command_executor.run_remote_command` given that it's a one-liner command

Reply:
In this way you can also avoid the usage of `install_extra_deps` in the cluster config file.

Reply:
In this case, I agree.